src/app/services/compute.service.ts

```diff
@@ -71,7 +75,65 @@ export class ComputeService {
     public listCustomImages(subscriptionId: string, location: string): Observable<Resource[]> {
         const params = new HttpParams()
             .set("$filter", `resourceType eq '${computeImageProvider}' and location eq '${location}'`);
```

Comment:
if both are used exactly the same you could always do a or query instead of 2 calls?

Reply:
I like that more

Reply:
I'll change

Reply:
the classics vs arm storage accounts listing use the same logic I believe

Reply:
if you want an example